### ISSUE TITLE
16.04 arm not working

### ISSUE THREAD

**Issue by delijati** [bhdouglass/clickable#48]
As mentioned on telegram `arm` build for 16.04 is not possible:

Error:
```
% docker run -v /opt/develop/privat/mobile/ubports/uMatriks:/opt/develop/privat/mobile/ubports/uMatriks -w /opt/develop/privat/mobile/ubports/uMatriks/build -u 1000 --rm -i clickable/ubuntu-sdk:16.04-armhf bash -c "qt5-qmake-arm-linux-gnueabihf  /opt/develop/privat/mobile/ubports/uMatriks" 
Could not find qmake configuration file .
Error processing project file: /opt/develop/privat/mobile/ubports/uMatriks/uMatriks.pro
```

With `-spec` it works
```
docker run -v /opt/develop/privat/mobile/ubports/uMatriks:/opt/develop/privat/mobile/ubports/uMatriks -w /opt/develop/privat/mobile/ubports/uMatriks/build -u 1000 --rm -i clickable/ubuntu-sdk:16.04-armhf bash -c "qt5-qmake-arm-linux-gnueabihf -spec /usr/lib/arm-linux-gnueabihf/qt5/mkspecs/ubuntu-arm-gnueabihf-g++  /opt/develop/privat/mobile/ubports/uMatriks"
```

Now the `make` command fails:
```
% docker run -v /opt/develop/privat/mobile/ubports/uMatriks:/opt/develop/privat/mobile/ubports/uMatriks -w /opt/develop/privat/mobile/ubports/uMatriks/build -u 1000 --rm -i clickable/ubuntu-sdk:15.04-armhf bash -c "make"    
cd uMatriks/ && ( test -e Makefile || /usr/bin/qt5-qmake-arm-linux-gnueabihf /opt/develop/privat/mobile/ubports/uMatriks/uMatriks/uMatriks.pro -spec /usr/lib/arm-linux-gnueabihf/qt5/mkspecs/ubuntu-arm-gnueabihf-g++ -o Makefile ) && make -f Makefile 
make[1]: Entering directory '/opt/develop/privat/mobile/ubports/uMatriks/build/uMatriks'
/usr/lib/qt5/bin/qmake -o Makefile ../../uMatriks/uMatriks.pro
make[1]: /usr/lib/qt5/bin/qmake: Command not found
Makefile:295: recipe for target 'Makefile' failed
make[1]: *** [Makefile] Error 127
make[1]: Leaving directory '/opt/develop/privat/mobile/ubports/uMatriks/build/uMatriks'
make: *** [sub-uMatriks-make_first] Error 2
Makefile:43: recipe for target 'sub-uMatriks-make_first' failed
```

**Comment by bhdouglass** [comment]
That's weird, /usr/lib/qt5/bin/qmake exists in the docker container. What's your clickable.json look like?

**Comment by delijati** [comment]
https://github.com/uMatriks/uMatriks/blob/master/clickable.json

**Comment by bhdouglass** [comment]
That's pretty standard, I'll have to dig deeper then.

**Comment by delijati** [comment]
I think @Flohack74 also had this problem in building telegram that is where i got the hint with [-spec](https://github.com/ubports/telegram-app/blob/bleeding-clickable/utils/build-click.sh#L8)

**Comment by Flohack74** [comment]
Ok I will try to do it with -spec, but I got this parameter in the build.sh I think. Need to take a look.

**Comment by bhdouglass** [comment]
I believe I've fixed this, you'll want to pull down the new docker image: `docker pull clickable/ubuntu-sdk:16.04-armhf`

**Comment by delijati** [comment]
I'm not sure if it is related to the `16.04` error but i get know this:

```
% docker run -v /opt/develop/privat/mobile/ubports/uMatriks:/opt/develop/privat/mobile/ubports/uMatriks -w /opt/develop/privat/mobile/ubports/uMatriks/build -u 1000 --rm -i clickable/ubuntu-sdk:16.04-armhf bash -c make -j
cd uMatriks/ && ( test -e Makefile || /usr/bin/qt5-qmake-arm-linux-gnueabihf -o Makefile /opt/develop/privat/mobile/ubports/uMatriks/uMatriks/uMatriks.pro ) && make -f Makefile 
make[1]: Entering directory '/opt/develop/privat/mobile/ubports/uMatriks/build/uMatriks'
make[1]: Nothing to be done for 'first'.
make[1]: Leaving directory '/opt/develop/privat/mobile/ubports/uMatriks/build/uMatriks'
cd lib/Matrix/ && ( test -e Makefile || /usr/bin/qt5-qmake-arm-linux-gnueabihf -o Makefile /opt/develop/privat/mobile/ubports/uMatriks/lib/Matrix/Matrix.pro ) && make -f Makefile 
make[1]: Entering directory '/opt/develop/privat/mobile/ubports/uMatriks/build/lib/Matrix'
make[1]: Nothing to be done for 'first'.
make[1]: Leaving directory '/opt/develop/privat/mobile/ubports/uMatriks/build/lib/Matrix'
cd /opt/develop/privat/mobile/ubports/uMatriks/po && LC_ALL=C intltool-update -d ca -g template -x && touch /opt/develop/privat/mobile/ubports/uMatriks/build/po/ca.po.updated
intltool-update: template.pot does not exist.
make: *** [/opt/develop/privat/mobile/ubports/uMatriks/build/po/ca.po.updated] Error 2
Makefile:374: recipe for target '/opt/develop/privat/mobile/ubports/uMatriks/build/po/ca.po.updated' failed
```

**Comment by bhdouglass** [comment]
@delijati I believe I've fixed the issue you & I were having. I was right that it was an issue in the ubuntu-click qmake extensions thingy, I've patched it in the 16.04-armhf docker image, you can pull it down to get the fix!
